utils/utils.go

```diff
@@ -447,3 +450,40 @@ func GetServiceConfig(name string, format string, value interface{}) (string, er
 
 	return buf.String(), nil
 }
+
+// ReadStrings reads string???
+func ReadStrings(comment string) (string, error) {
```

Comment:
I think should change to ReadUserInput()?

Reply:
And this function just read a line, not multiple lines, so ReadString**s** is confusing

Reply:
tbh i don't even understand the purpose of this function